packages/cli/src/lib/cmds/play.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {
+  Config, GradleWrapper, JdkHelper, AndroidSdkTools, ConsoleLog, Log, GooglePlay, TwaManifest,
+} from '@bubblewrap/core';
+import {Track} from '@bubblewrap/core/dist/lib/GooglePlay';
```

Comment:
This import doesn't look right. We probably want something like:

`import {Track} from `@bubblewrap/core`.

You'll probably need to modify `index.ts` in core for this.

Reply:
Done. Good Catch!